samples/api-client/accesstoken_example/accesstoken.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
```

Comment:
would be nice to have a demo flow with access token generation and pubsub publish+storage upload.